lib/nvme/nvme_qpair.c

```diff
@@ -678,6 +678,22 @@ static int _nvme_qpair_build_prps_sgl_request(struct nvme_qpair *qpair,
 			return -1;
 		}
 
+		if ((phys_addr & 0x3) != 0) {
+			// address must be dword aligned
```

Comment:
Looks good. Just one nit: Can you move the comment right above the "if ()" since it explains what is being tested. And also please use C style comments "/* */". Same for the 2 other comments/ifs below this one.